python/dolfinx/io/utils.py

```diff
@@ -147,15 +147,25 @@ def __exit__(self, exception_type, exception_value, traceback):
         self.close()
 
     def write_mesh(self, mesh: Mesh, xpath: str = "/Xdmf/Domain") -> None:
-        """Write mesh to file for a given time (default 0.0)"""
+        """Write mesh to file"""
         super().write_mesh(mesh._cpp_object, xpath)
 
     def write_meshtags(self, tags: MeshTags, geometry_xpath: str = "/Xdmf/Domain/Grid/Geometry",
                        xpath: str = "/Xdmf/Domain") -> None:
-        """Write mesh tags to file for a given time (default 0.0)"""
+        """Write mesh tags to file"""
         super().write_meshtags(tags._cpp_object, geometry_xpath, xpath)
 
     def write_function(self, u, t: float = 0.0, mesh_xpath="/Xdmf/Domain/Grid[@GridType='Uniform'][1]"):
+        """
+        Write function to file for a given time.
+
+        NOTE: Function is interpolated onto the mesh nodes, as either a Nth order Lagrange function or DG-0 function.
```

Comment:
Not Google docstring style, see https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html.

The note isn't clear to me. When is it interpolate to a Lagarnge function of degree `n`, what is `n`, and when will it be DG-0 (which would better stated as cell-wise).